lib/src/Uncertainty/Distribution/openturns/DistFunc.hxx

```diff
@@ -295,8 +301,20 @@ OT_API Sample rUniformTriangle(const Point & a,
                                const Point & b,
                                const Point & c,
                                const UnsignedInteger size);
-// For TruncatedNormal distribution
-//      Scalar rTruncatedNormal(const Scalar a, const Scalar b);
+// For uniform distribution over a tetrahedron
+OT_API Point rUniformTetrahedron(const Point & a,
+                                 const Point & b,
+                                 const Point & c,
+                                 const Point & cd);
+OT_API Sample rUniformTetrahedron(const Point & a,
+                                  const Point & b,
+                                  const Point & c,
+                                  const Point & d,
+                                  const UnsignedInteger size);
+// For uniform distribution over a simplex
+OT_API Point rUniformSimplex(const Sample & vertices);
+OT_API Sample rUniformSimplex(const Sample & vertices,
+                              const UnsignedInteger size);
```

Comment:
the new SpecFunc, DistFunc methods must be listed in the usermanual

Reply:
@jschueller The documentation in commit ff7f62c136c4c57137cff0cceedd3183dc9eb0a6 does not fit your expectations?

Reply:
mostly :)
but the SpecFunc functions must be explicitely listed:
https://github.com/openturns/openturns/blob/master/python/doc/user_manual/functions.rst
same for DistFunc methods in probabilistic_modelling.rst

Reply:
I got it! For SpecFunc, it was just an extension of existing functions to the complex domain so they were already in the User Manual. I just added the new functions proposed by DistFunc.